deploy/docker/tachyon-dev/tachyon-base/files/log4j.properties

```diff
@@ -19,6 +19,17 @@ log4j.appender.MASTER_LOGGER.layout=org.apache.log4j.PatternLayout
 log4j.appender.MASTER_LOGGER.layout.ConversionPattern=%d{ISO8601} %-5p %c{2} (%F:%M) - %m%n
 #log4j.appender.MASTER_LOGGER.layout.ConversionPattern=%d{ISO8601} %-5p %c{2} (%F:%M(%L)) - %m%n
 
+# Appender for Master Access Log
```

Comment:
It seems like the docker logging config is slightly different from the project logging config. Do you know why that is?